config/locales/devise/en.yml

```diff
@@ -9,20 +9,20 @@ en:
       admin:
         already_authenticated: You are already signed in.
         inactive: Your account is not activated yet.
-        invalid: The password or admin user name you entered doesn't match our records.
+        invalid: The password or admin username you entered doesn't match our records.
         last_attempt: You have one more attempt before your account is locked.
         locked: Your account is locked.
-        not_found_in_database: The password or admin user name you entered doesn't match our records.
+        not_found_in_database: The password or admin username you entered doesn't match our records.
         timeout: Your session expired. Please sign in again to continue.
         unauthenticated: You need to sign in or sign up before continuing.
         unconfirmed: You have to confirm your email address before continuing.
       user:
         already_authenticated: You are already signed in.
         inactive: You'll need to activate your account before you can log in. Please check your email or contact support.
-        invalid: The password or user name you entered doesn't match our records. Please try again or <a href="%{reset_path}">reset your password</a>. If you still can't log in, please visit <a href="%{problems_path}">Problems When Logging In</a> for help.
+        invalid: The password or username you entered doesn't match our records. Please try again or <a href="%{reset_path}">reset your password</a>. If you still can't log in, please visit <a href="%{problems_path}">Problems When Logging In</a> for help.
```

Comment:
I was going to ask why on earth this isn't coded like our other links, but I see it's an existing issue and Devise, so! We'll just ignore that, I guess. (And why doesn't this key need `_html`?)

Reply:
Yes, I had the same thought on seeing it :sob: